### PR TITLE
docs: Added instructions for installation with pnpm

### LINKED ISSUE #19918: Instructions for installing through pnpm are missing
### Documentation issue

- [ ] Reporting a typo
- [ ] Reporting a documentation bug
- [x] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?

All the overview pages of API's cards, like react, next, nuxt, express, etc. 

### Additional context or description

- [ ] The layout is not the same for all these pages, some are in tabs, and others are in a single box or multiple boxes.
- [ ] Instructions for installing using pnpm is missing

### PR DESCRIPTION
**Pull request by MathurAditya724**
## Current Behavior 
- Mixed layout for install instructions (Tabs, separate code blocks, or single code block)
- pnpm instructions were not present on all the API pages

## Expected Behavior
- Simple and similar layouts
- Instructions for pnpm

## Related Issue(s)

Fixes #19918

### CODE REVIEW

**Comment by MathurAditya724** [comment]
Hello guys. any update on this?

File: docs/generated/packages/angular/documents/overview.md [hunk 1]
```diff
@@ -25,14 +25,29 @@ can [fix Nx version mismatches with this recipe](/recipes/tips-n-tricks/keep-nx-
...
+```shell
+npm install --save-dev @nx/angular
+```
+
+{% /tab %}
+{%tab label="yarn"%}
+
 ```shell
-yarn add -D @nx/angular
+yarn add --dev @nx/angular
```
Comment (meeroslav) [inline docs/generated/packages/angular/documents/overview.md:39]: Why do you think this change was necessary?

File: docs/generated/packages/angular/documents/overview.md [hunk 2]
```diff
@@ -25,14 +25,29 @@ can [fix Nx version mismatches with this recipe](/recipes/tips-n-tricks/keep-nx-
...
 ```
 
+{% /tab %}
+{%tab label="pnpm"%}
+
 ```shell
-npm install -D @nx/angular
+pnpm install --dev @nx/angular
```
Comment (meeroslav) [inline docs/generated/packages/angular/documents/overview.md:46]: Why do you think this change was necessary?

**Comment by meeroslav** [comment]
Thank you for the PR.

For the sake of making targeted changes, can you limit your changes to just adding `pnpm` where `yarn` and `npm` exist without changing the short signature of the existing commands?

**Comment by MathurAditya724** [comment]
Sure, I will do that

File: docs/generated/packages/jest/documents/overview.md [hunk 3]
```diff
@@ -22,14 +22,30 @@ First, install `@nx/jest`, if not already installed using your preferred package
...
+
 ```shell
 yarn add --dev @nx/jest
 ```
 
+{% /tab %}
+{%tab label="pnpm"%}
+
+```shell
+pnpm install --dev @nx/jest
```
Comment (barbados-clemens) [inline docs/generated/packages/jest/documents/overview.md:43]: why use the short hand of npm install but not pnpm?
we should probably be consistent 
this applies to a lot of the spots from what I can tell.

**Comment by MathurAditya724** [comment]
Sorry for closing this, needed to do it again from scratch. I have created a new PR for this #21226

File: docs/generated/packages/jest/documents/overview.md (hunk 3 above)
Comment (MathurAditya724) [reply docs/generated/packages/jest/documents/overview.md]: I have completed these changes in this new PR #21226

**Pull request closed without merging**